docs/extensions/lifecycle.md

```diff
@@ -248,6 +248,22 @@ Based on a project's maturity and stewardship, it may be eligible to use resourc
 | QA	| The `civicrm.org` build-bot runs extension tests for PRs (civicrm-core repo)	| "Official" projects ("Stable" or "Incubation")
 | Support	| The project may have its own space or component on "lab.civicrm.org"	| "Official" projects (regardless of stability)
 
+## Obsolete Extensions
+
+Sometimes an extension's functionality becomes part of CiviCRM Core, or is otherwise redundant, and the extension *should no longer be used*.
+
+At that point it can be added to the [Extension Compatability List](https://github.com/civicrm/civicrm-core/blob/master/extension-compatibility.json)
```

Comment:
```suggestion
At that point it can be added to the [Extension Compatibility List](https://github.com/civicrm/civicrm-core/blob/master/extension-compatibility.json)
```